docs/tutorial/part-zero/index.md

```diff
@@ -1,325 +1,330 @@
 ---
-title: Set Up Your Development Environment
+title: Pystytä sinun kehitysympäristö
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Before you start building your first Gatsby site, you’ll need to familiarize yourself with some core web technologies and make sure that you have installed all required software tools.
+Ennen kuin aloitat sinun ensimmäisen Gatsby-sivuston rakentamisen, sinun on perehdyttävä joihinkin ydin web-teknologioihin ja varmistettava, että olet asentanut kaikki vaadittavat ohjelmistotyökalut.
 
-## Familiarize yourself with the command line
+## Tutustu komentoriviin
 
-The command line is a text-based interface used to run commands on your computer. You’ll also often see it referred to as the terminal. In this tutorial, we’ll use both interchangeably. It’s a lot like using the Finder on a Mac or Explorer on Windows. Finder and Explorer are examples of graphical user interfaces (GUI). The command line is a powerful, text-based way to interact with your computer.
+Komentorivi on tekstipohjainen käyttöliittymä, jota käytetään komentojen suorittamiseen tietokoneellasi. Saatat myös nähdä, että sitä kutsutaan terminaaliksi. Käytämme tässä tutoriaalissa molempia keskenään vaihtokelpoisesti.
+Se on paljon kuin Finderin käyttö Macissa tai Resurssienhallinta Windowsissa. Finder ja Explorer ovat esimerkkejä graafisista käyttöliittymistä (GUI). Komentorivi on tehokas, tekstipohjainen tapa olla vuorovaikutuksessa tietokoneesi kanssa.
 
-Take a moment to locate and open up the command line interface (CLI) for your computer. Depending on which operating system you are using, see [**instructions for Mac**](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/), [**instructions for Windows**](https://www.lifewire.com/how-to-open-command-prompt-2618089) or [**instructions for Linux**](https://www.howtogeek.com/140679/beginner-geek-how-to-start-using-the-linux-terminal/).
+_**Huom:** Kaikki ohjeet ovat englanniksi._
```

Comment:
Maybe we should remove this line from here because it messes up the line numbering and might cause issues when there are incoming updates from the English repo. Also makes the review harder because the original and translation do not appear side by side.

Reply:
Ok 👍